### ISSUE TITLE
False positive W002

### ISSUE THREAD

**Issue by freezy-sk** [twbs/bootlint#338]
I'm getting warning `W002 `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes` even when there is `<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />`

**Comment by tschortsch** [comment]
It seems that the following code snippet just allows "IE=edge" as content:
https://github.com/twbs/bootlint/blob/master/src/bootlint.js#L332

The meta-Array should look like this:
```javascript
var meta = $([
    'head>meta[http-equiv="X-UA-Compatible"][content="IE=edge"]',
    'head>meta[http-equiv="x-ua-compatible"][content="ie=edge"]',
    'head>meta[http-equiv="x-ua-compatible"][content="IE=edge,chrome=1"]',
    'head>meta[http-equiv="x-ua-compatible"][content="ie=edge,chrome=1"]'
].join(','));
```

**Comment by tschortsch** [comment]
It seems that the chrome=1 information is not used anymore because the Google Chrome Frame was discontinued in January 2014. See: http://stackoverflow.com/a/22059516.
So you might just remove this part from you meta-Tag:
```
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

**Comment by zacechola** [comment]
Duplicate of #302 and #152. In the past we've closed this issue, but I'll leave it open for now since there's a related PR.

**Comment by cvrebert** [comment]
Closing per #152. The Chrome Frame project is dead and hasn't been updated in nearly 2 years.
